Projects/ChangeTier/Sources/Views/ChangeTierSummaryScreen.swift

```diff
@@ -0,0 +1,51 @@
+import SwiftUI
+import hCore
+import hCoreUI
+
+struct ChangeTierSummaryScreen: View {
+    @ObservedObject var vm: ChangeTierViewModel
+    @EnvironmentObject var changeTierNavigationVm: ChangeTierNavigationViewModel
+
+    var body: some View {
+        let displayItems: [QuoteDisplayItem] =
+            vm.selectedTier?.displayItems.map({ .init(title: $0.title, value: $0.value) }) ?? []
+
+        let vm = QuoteSummaryViewModel(
```

Comment:
Can we extract this from here?
Like 
```
struct ChangeTierSummaryScreen: View {
    @ObservedObject var vm: ChangeTierViewModel
    @EnvironmentObject var changeTierNavigationVm: ChangeTierNavigationViewModel
    let quoteSummaryVm: QuoteSummaryViewModel

   init(vm: ChangeTierViewModel) {
         self.vm = vm
         quoteSummaryVm = vm.asQuoteSummaryViewModel()
....
```

and later on:
```
extension on ChangeTierViewModel {
   var asQuoteSummaryViewModel() - > QuoteSummaryViewModel {
          //do the magic
   }
}
```

Reply:
Yes that's much cleaner